chainio/clients/avsregistry/reader.go

```diff
@@ -361,17 +361,26 @@ func (r *ChainReader) QueryRegistrationDetail(
 ) ([]bool, error) {
 	operatorId, err := r.GetOperatorId(opts, operatorAddress)
 	if err != nil {
-		return nil, err
+		return nil, utils.WrapError("Failed to get operator id", err)
 	}
 	value, err := r.registryCoordinator.GetCurrentQuorumBitmap(opts, operatorId)
 	if err != nil {
-		return nil, err
+		return nil, utils.WrapError("Failed to get operator quorums", err)
 	}
 	numBits := value.BitLen()
 	var quorums []bool
 	for i := 0; i < numBits; i++ {
 		quorums = append(quorums, value.Int64()&(1<<i) != 0)
 	}
+	if len(quorums) == 0 {
```

Comment:
can you add a comment explaining when/why this case is possible? When does GetCurrentQuorumBitmap return a value of 0 bitlen?

Reply:
I think if operator has opt out from all quorums (or been ejected), the contract could return 0, you can see here:
![Uploading image.png…]()

Reply:
I think if operator has opt out from all quorums (or been ejected), the contract could return 0, you can see here:
![image](https://github.com/user-attachments/assets/f5ee4ec2-570f-4189-b01e-bb499527f358)

Reply:
meant in the code, because it's surprising when reading the code why it could ever be 0.